src/core/components/App.tsx

```diff
@@ -41,6 +43,19 @@ const AppRoutes: React.FC = (): JSX.Element => {
         }
     }, [window.location.href]);
 
+    // useEffect(() => {
```

Comment:
can remove comment

Reply:
Opps

Reply:
I committed the code but forgot to push. Thanks for catching.